about.html

```diff
@@ -0,0 +1,114 @@
+          <li class="menu-item"><a href="#">Program</a></li>
+          <li class="menu-item"><a href="#">Join</a></li>
+          <li class="menu-item"><a href="#">Sponsor</a></li>
+          <li class="menu-item"><a href="#">News</a></li>
+        </ul>
+      </div>
+    </div>
+
+    </div>
+  </div>
+
+  <div class="main-wrap wrap">
+    <section class="about-section">
+      <h1 class="normal about">About</h1>
+      <div class="guide-bar guide-about"></div>
+      <div class="message">
+        <p>Each year, 8,000+ developers, engineers, software architects, dev teams, managers, and executives from 70+ countries gather for DeveloperWeek (Feb 15–17, San Francisco Bay Area & Feb 21-23, Virtual) to discover the latest in developer technologies, languages, platforms, and tools. </p>
+        <p>DeveloperWeek 2023 is the largest developer conference & event series including the DeveloperWeek 2023 Conference & Expo, 1,000+ attendee hackathon, 800+ attendee tech hiring expo, and a series of workshops, technical talks, and keynotes.</p> 
+        <p>Because DeveloperWeek covers all new technologies, our conference and workshops invite you to get intro lessons (or advanced tips and tricks) on technologies like VR Dev, Artificial Intelligence Dev, Blockchain Dev, IoT Dev, Serverless technology, Microservices technology, new JavaScript frameworks, and more.</p>
+        <p>This year, DeveloperWeek will have an expanded DevExec World, which is 2 days of roundtable talks, educational talks, and networking events for engineering managers, tech executives, and lead developers. Additionally, DeveloperWeek will be co-located with ProductWorld and CloudWorld.</p>
+        <p>DeveloperWeek puts the spotlight on new technologies. Companies that participated in past DeveloperWeek events include Google, Facebook, Yelp, Rackspace, IBM, Cloudera, Red Hat, Optimizely, SendGrid, Blackberry, Microsoft, Neo Technology, Eventbrite, Klout, Built.io, Ripple, GNIP, Tagged, HackReactor, and 30+ more here!</p>
+        <p>When it comes to technology, there’s incremental change, and then there’s fundamental innovation. Developer technology, from blockchain and artificial intelligence to big data and quantum computing represents fundamental innovation that people can build on for years. We are in the DevTech Age, where developer technologies and tools are now the most disruptive and fundamental technology innovation in the marketplace. When you build tools for developers, you are not just implementing a small incremental use case, you are building platforms, frameworks, and APIs that will enable entirely new web, mobile, and IoT innovation.</p>
+      </div>
+      <div class="location-about">
+        <div class="dateinfo">LEARN. CONNECT. EXPERIENCE.</div>
+        <div class="locate">@ DevNetwork. All rights reserved.</div>
+      </div>
+    </section>
+  </div>
+
+  <footer class="site-footer">
+    <div class="wrap footer-black">
+      <div class="flex column">
```

Comment:
- [ ] Kindly ensure that your about section looks like the design template 👍🏽 

Design template 👇🏽  

![image](https://user-images.githubusercontent.com/63473584/197054609-d5f565ee-e44d-4802-b4dc-bddffba4c568.png)

Your Design 👇🏽 

![image](https://user-images.githubusercontent.com/63473584/197054880-cd8157c6-969e-4e03-a232-d2d040f7d417.png)